### ISSUE TITLE
Add a proper type for Sport

### ISSUE THREAD

**Issue by vholland** [schemaorg/schemaorg#2401]
(This is branching off of issue #1951 )

The https://schema.org/sport property takes Text or a URL. It would be nice to have a proper https://schema.org/Sport type so we can assign sports names and sameAs links. We could even be able to say what is called 'football' in USA is not 'football' as the rest of the world understands it!

Ideally, we would not stand on ceremony and would allow the type to be named 'Sport' despite already having a property of the same name.

**Comment by KenSharp** [comment]
+1

`sameAs` would be extremely useful when disambiguating between rules, especially in different countries.

“Rugby Union”, for example, has different rules depending on age groups. This can perhaps be inferred by the age grouping of the specific teams.

“Rugby League” on the other hand has different rules depending on which country you're talking about, and you would probably want to know which rules a team is playing by during international fixtures.

There may also be different rules depending on gender in some countries.

I've just butt up against this issue when trying to define which flavour I am referring to on my website. All I have is `"sport": "Rugby Union"` which is a little too vague.

**Comment by espinr** [comment]
In case anyone finds it interesting, the Open Active CG defined [a sports taxonomy](https://activity-list.openactive.io/en/hierarchical_concepts.html). In the Open Track CG, we defined a more fine-grained [model and taxonomy focused on athletics](https://w3c.github.io/opentrack-cg/spec/model/#sports-discipline).

**Comment by DaveyJake** [comment]
> "Rugby Union", for example, has different rules depending on age groups. This can perhaps be inferred by the age grouping of the specific teams.

@KenSharp Nice to see **Rugby** getting more discussion! I've been obsessed trying to figure out a way to make [SportsML + SchemaOrg](https://github.com/schemaorg/suggestions-questions-brainstorming/issues/163) a reality.

> I've just butt up against this issue when trying to define which flavour I am referring to on my website. All I have is `"sport": "Rugby Union"` which is a little too vague.

I used to work at USA Rugby.  When I wrote the original USAR schema, I defined `sport` using the Wikidata URI for [Rugby Union](https://www.wikidata.org/wiki/Q5849).  It helps across all search engines when scoring for relativity.

`"sport": "https://www.wikidata.org/wiki/Q5849"`

**Comment by KenSharp** [comment]
> "sport": "https://www.wikidata.org/wiki/Q5849"

I didn't realise that a URL was an option. Interesting.

**Comment by DaveyJake** [comment]
> I didn't realise that a URL was an option. Interesting.

That's right, mate.  As it says on the [sport](https://schema.org/sport) page...

> Values expected to be one of these types: `Text`, `URL`

**Comment by bquinn** [comment]
> @KenSharp Nice to see **Rugby** getting more discussion! I've been obsessed trying to figure out a way to make [SportsML + SchemaOrg](https://github.com/schemaorg/suggestions-questions-brainstorming/issues/163) a reality.

In that case, you might like to see our IPTC work in progress at https://sportschema.org/ ...

**Comment by DaveyJake** [comment]
> In that case, you might like to see our IPTC work in progress at https://sportschema.org/ ...

FINALLY!!! Can't wait to dig into this!
